stanify/calibrator/calib_utilities.py

```diff
@@ -0,0 +1,13 @@
+
+def check_consistency(setting, precision, numeric, prior, output_format):
+    is_consistent = True
+    #TODO @Dashadowe better error msg format?
```

Comment:
> better error msg format?

I would raise an exception if it's fatal.

Reply:
Automating to the level of there is no need for consistency check is our goal.